services/core-api/app/api/mines/permits/permit/resources/permit.py

```diff
@@ -111,6 +128,24 @@ def post(self, mine_guid):
         db.session.add(permit)
         db.session.add(amendment)
 
+        now_application_guid = data.get('now_application_guid')
```

Comment:
I think we should raise a Bad Request error if `now_application_guid` is not provided.

Reply:
This endpoint works without a now_guid - should we still throw a 'Bad Request error` given that the majority of the POSTS do not have a now_guid?

Reply:
I didn't realize that. If it's an optional parameter then it's all good!